examples/data-objects/prosemirror/src/prompt.ts

```diff
@@ -146,13 +147,14 @@ export class Field {
     validateType(_value): string | undefined { return; }
 
     validate(value) {
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
         if (!value && this.options.required) { return "Required field"; }
         // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return this.validateType(value) || this.options.validate?.(value);
+        return Boolean(this.validateType(value)) || this.options.validate?.(value);
```

Comment:
validateType always returns undefined above?  Maybe just remove it entirely

Reply:
Good catch. Fixed.